index.html

```diff
@@ -4240,47 +4240,54 @@ <h2>HTTP</h2>
         </section>
 
         <section class="informative">
-          <h2>JSON Processing</h2>
-
+          <h2>JSON-LD Processing</h2>
           <p>
 While the media types describing conforming documents defined in this
-specification always express JSON-LD, JSON-LD processing is not required to be
-performed, since JSON-LD is JSON. Some scenarios where processing a
-<a>verifiable credential</a> or a <a>verifiable presentation</a> as JSON is
+specification always express JSON-LD, RDF processing is not required to be
+performed, since JSON-LD is a concrete RDF syntax as described in [RDF11-CONCEPTS]. 
+Hence, a JSON-LD document is both an RDF document and a JSON document and correspondingly represents an instance of an RDF data model.
+See <a href="JSON-LD11##relationship-to-rdf">Relationship to RDF</a> for more details.
+          </p>
+          <p>
+Some scenarios where processing a <a>verifiable credential</a> or a <a>verifiable presentation</a> as RDF is
 desirable include, but are not limited to:
           </p>
-
           <ul>
             <li>
-Before securing or after verifying content
-that requires <a href="https://csrc.nist.gov/glossary/term/data_integrity">data
-integrity</a>, such as a
-<a>verifiable credential</a> or <a>verifiable presentation</a>.
+Before securing or after verifying <a>verifiable credentials</a> or <a>verifiable presentations</a>,
+in cases where the issuer has leveraged the <code>@context</code> to provide details regarding 
+their intended expression of the <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>.
             </li>
             <li>
-When performing JSON Schema validation, as described in Section
-<a href="#data-schemas"></a>.
+After performing JSON Schema validation, as described in Section
+<a href="#data-schemas"></a>, in order to mitigate attacks 
+on the canonicalization process as described in <a href="https://w3c.github.io/rdf-canon/spec/#dataset-poisoning">Dataset Poisoning</a>
             </li>
             <li>
 When serializing or deserializing <a>verifiable credentials</a> or
-<a>verifiable presentations</a> into systems that store or index their contents.
+<a>verifiable presentations</a> into systems that store or index claims about subject in
+<code>text/turtle</code>, <code>application/n-quads</code>, <code>application/ld+json</code>. 
```

Comment:
Systems do not store RDF datasets in text/turtle and friends. Those are all serialization formats. RDF runtimes like RDFLib, rdf-js, Jena, etc, or RDF Databases like Virtuoso or Amazon Neptune all store the data in their propriatery format.

```suggestion
<a>verifiable presentations</a> into an RDF Data Store. 
```

B.t.w., the counterpart of n-quads should be text/trig and not text/turtle

Reply:
@iherman I applied Manu's suggestion, which I thing better addresses this

Reply:
I do not see any change. The text still says 

> When serializing or deserializing [verifiable credentials](http://localhost:8001/LocalData/github/VC/vc-data-model/#dfn-verifiable-credential) or [verifiable presentations](http://localhost:8001/LocalData/github/VC/vc-data-model/#dfn-verifiable-presentation) into systems that store or index data in Turtle [[TURTLE](http://localhost:8001/LocalData/github/VC/vc-data-model/#bib-turtle)], N-Quads [[N-QUADS](http://localhost:8001/LocalData/github/VC/vc-data-model/#bib-n-quads)], or JSON-LD [[JSON-LD11](http://localhost:8001/LocalData/github/VC/vc-data-model/#bib-json-ld11)].

which is what I objected to: most of the (RDF) data does not store their data in serialized form. With that exclusion the list item is meaningless...